<!-- script for waypoint flight in offboard mode -->
<launch>
    <!-- number of points, be precise -->
    <param name="point_num" value="3"  />

    <!-- way points -->
    <!-- mind param's name with id from 0 to point_num-1 -->
    <param name="point0_x" value="0.0"  />
    <param name="point0_y" value="0.0"  />
    <param name="point0_z" value="4.0"  />

    <param name="point1_x" value="10.0" />
    <param name="point1_y" value="0"    />
    <param name="point1_z" value="8.0"  />

    <param name="point2_x" value="-10.0" />
    <param name="point2_y" value="5.0"   />
    <param name="point2_z" value="4.0"   />

    <node pkg="px4_mavros_controller"
        type="offb_node" name="offb_node_debug" output="screen">
    </node>

</launch>